brainflow-dev/brainflow
# add support for mentalab devices

Signed-off-by: Andrey Parfenov <a1994ndrey@gmail.com>

Tested on windows, can connect to device and get data.

Todo and issues:
- [x] can connect to device only when LED is blue and blinking, after some time it gets green and you have to restart the device
- [x] temperature data looks completely wrong, its a random value between 20 and 120, need to check parsing
- [ ] accel and gyro data, instead expected values like -1..0..1(these units used in brainflow) I get something I dont understand, the same in explorepy, so its not an issue with parsing
- [x] add docs about explore 4 and 8 channels boards
- [x] add release notes to website
- [ ] need to test on linux and macos, have no hardware
- [ ] cannot test correctness of eeg parsing(I dont get how to connect these electrodes), dont understand the idea of 5th channel... is it a reference which should be subtracted from other channels? these value does not change for me 
- [x] it seems like fw supports timestamp syncronization, need to implement it or at least use sampling rate to correct timestamps which come from batches of datapoints
- [ ] need to test 8 chan board, have no device 

To do later in case of requests:

- add support for sending commands(at least for sampling rate and maybe on\off)
- add support for impedance measurement
